website/static/js/osfHelpers.js

```diff
@@ -774,6 +774,12 @@ var confirmDangerousAction = function (options) {
     bootbox.dialog(bootboxOptions);
 };
 
+/** A future-proof getter for the current user
+**/
+var currentUser = function(){
+    return window.contextVars.currentUser;
```

Comment:
I feel that if the current user is not defined, then this function should return undefined.